extensions/interactions/NumericInput/directives/oppia-interactive-numeric-input.component.ts

```diff
@@ -41,7 +41,7 @@ export class InteractiveNumericInput implements OnInit {
   errorString = '';
   requireNonnegativeInput: boolean = false;
   answer = null;
-  NUMERIC_INPUT_FORM_SCHEMA: { type: string; 'ui_config': {}; };
+  NUMERIC_INPUT_FORM_SCHEMA: { type: string; 'ui_config': {} };
```

Comment:
As the same type is used below can you put it in a type variable and use it.

Reply:
Done